src/system/components/Icon/icons/type.ts

```diff
@@ -0,0 +1,4 @@
+export interface IconBaseType {
+  color?: string;
+  size?: number;
+}
```

Comment:
```suggestion
export type IconType = SVGProps<SVGSVGElement> & { color?: string; size?: nubmer };
```

위와 같이 사용하는건 어떻게 생각하시는지 궁금합니다 !
svg 태그의 attribute를 그대로 사용할 수 있습니다 !

Reply:
요거 고민하다가 우선 확장하지는 않았어요. icon을 div로 한 번 감쌀 필요는 없을까? svg를 그대로 export하는 형태가 맞을까? 라는 고민을 하다가 `추가하기는 쉽지만, 제거하기는 어렵다`는 개인적인 생각으로 우선은 제한해보았어요!